pytest.ini

```diff
@@ -28,6 +28,7 @@ markers =
     snowflake: test for Snowflake
     spark: test for Spark
     trino: test for Trino
+addopts = -n auto --dist=loadscope
```

Comment:
@izeigerman `-n <X>` is the same as `--tx <X>*popen//python=python` as long as the Python interpreter you are targeting is the same as what you are running Pytest with. In other words, `--tx` is for assigning work to another python interpreter and you can shortcut this with `-n` is you are running on the same interpreter.

Reply:
Beautiful, thanks!